plugins/bright_plugin/src/bright-ui/src/app/bright/packages/time-picker-modal/time-picker-modal.constants.ts

```diff
@@ -0,0 +1,32 @@
+export class TimePickerModalConstants {
+    public static TIME_RANGES = [
+        {label: 'Last 5 Minutes', value: 'Last 5 Minutes', serverKey: '5'},
+        {label: 'Last 1 hour', value: 'Last 1 hour', serverKey: '60'},
+        {label: 'Last 12 hours', value: 'Last 12 hours', serverKey: '720'},
+        {label: 'Last 24 hours', value: 'Last 24 hours', serverKey: '1440'},
+        {label: 'Last week', value: 'Last week', serverKey: '10080'},
+        {label: 'Last month', value: 'Last month', serverKey: '43200'},
+        {label: 'Last 6 months', value: 'Last 6 months', serverKey: '259200'},
+        {label: 'Last 1 year', value: 'Last 1 year', serverKey: '525600'}
```

Comment:
this is not last year, it's last 12 months! 
also the whole calculations of months and years can't be multiply of minutes, since months could be 28,29,30 or 31 days long 
but let me get back to you for this issue.